lib/boku/tests/utils.py

```diff
@@ -71,14 +73,16 @@ def add_seller_boku(self):
                                                      service_id='456')
 
     def sample(self):
-        return {
+        data = {
             'action': 'billingresult',
             'amount': '100',
             'currency': 'MXN',
             'param': 'some:uuid',
-            'sig': 'some:sig',
             'trx-id': 'some:trxid'
         }
+        data['sig'] = get_boku_request_signature(
+            settings.BOKU_SECRET_KEY, get_pairs_string(data))
```

Comment:
I think ``solitude/settings/test.py`` should set a value for ``BOKU_SECRET_KEY`` for this.